README.md

```diff
@@ -5,368 +5,17 @@ https://www.simplereport.gov/
-
-#### Notes
-Make sure docker has access to enough resources on your local machine. You'll know it doesn't if you get an error from any of your containers similar to this:
-```
-"The build failed because the process exited too early. This probably means the system ran out of memory or someone called `kill -9` on the process."
-```
-
-How to update resources limits on your [Mac](https://docs.docker.com/desktop/mac/#resources), and [Windows](https://docs.docker.com/desktop/windows/#resources) machines.
-
-## Running outside of docker
-
-If you wish to run outside of docker compose (bare metal), you'll need to install all the dependencies locally.
-See more [info here](https://github.com/CDCgov/prime-simplereport/wiki/Running-outside-of-docker).
-
-## Roles
-
-The available user role types are `ADMIN`, `USER`, `ENTRY_ONLY`, `ALL_FACILITIES`, and `NO_ACCESS`. You can check `backend/src/main/java/gov/cdc/usds/simplereport/config/authorization/OrganizationRole.java` for a list of available roles
-
-- `ADMIN` - an organization admin with full access to their organization
-- `USER` - a site user the has access to everything in their organization but the gear icon
-- `ENTRY_ONLY` - a site user that only has access to the Conduct Test tab
-- `ALL_FACILITIES` - a site user that can access all facilities in their organization
-- `NO_ACCESS` - a member of an organization who has no permissions without possessing other roles. Every member of an org has this role, so it is used to list all users in an organization
-
-These roles are controlled via Okta groups.
-
-## Support admin
-
-You can make your user a support admin by assigning yourself the `SR-DEV-ADMINS` group in Okta Preview.
-
-Support admins can access the `/admin` paths and support admin APIs.
-
-## Database cleaning
-
-When there are DB schema changes the backend may throw an error and fail to start.
-
-To create a fresh database:
-
-1. `docker compose down --volumes`
-
-Note that until we create seed data, this will start you with an empty database, and you will need to manually create organizations and facilities again.
-
-## Rollbacks
-
-The application uses the Liquibase plugin for Gradle to perform certain database management tasks.
-
-To roll the database back to its state at a prior date:
-
-```
-docker compose run --rm backend gradle liquibaseRollbackToDate -PliquibaseCommandValue=${date}
-```
-
-To roll back a certain _number_ of migrations:
-
-```
-docker compose run --rm backend gradle liquibaseRollbackCount -PliquibaseCommandValue=${n}
-```
-
-To roll back to a certain tag:
-
-```
-docker compose run --rm backend gradle liquibaseRollback -PliquibaseCommandValue=${TAG}
-```
-
-If you are required to roll back a non-local database, you may generate the required SQL to execute elsewhere. Use `liquibaseRollbackToDateSQL` or `liquibaseRollbackCountSQL` in the manner described above to write the rollback SQL to stdout.
-
-## API testing
```

Comment:
⚠️ Moved to https://github.com/CDCgov/prime-simplereport/wiki/API-Testing-with-Insomnia